whois-commons/src/main/resources/internals_schema.sql

```diff
@@ -1,12 +1,3 @@
-DROP TABLE IF EXISTS `scheduler`;
```

Comment:
Create a separate patch script for removing the scheduler table, after we've deployed the release

Reply:
I'll add a task to the backlog to remove the table seperately in the next release just in case we have to rollback or something